src/GraphQLMiddleware.php

```diff
@@ -67,12 +61,7 @@ public function __invoke(ServerRequestInterface $request, ResponseInterface $res
     }
 
     private function isGraphQLRequest(ServerRequestInterface $request) {
-        return $this->hasUri($request) || $this->hasGraphQLHeader($request);
```

Comment:
The right place for this is in the config of the pipeline, cf. ModuleConfig.php ("path" key) :
```
            "middleware_pipeline" => [
                [
                    'middleware' => [
                        GraphQLMiddleware::class
                    ],
                    'path' => '/api/graphql',
                    'priority' => 1000,
                ],
            ]
```
See comments in expressive skeleton's [middleware-pipeline.global.php](https://github.com/zendframework/zend-expressive-skeleton/blob/master/config/autoload/middleware-pipeline.global.php#L21)